ocw_data_parser/ocw_data_parser.py

```diff
@@ -490,13 +507,13 @@ def generate_parsed_json(self):
             "title": self.jsons[0].get("title"),
             "description": self.jsons[1].get("description"),
             "other_information_text": self.jsons[1].get("other_information_text"),
-            "first_published_to_production": self.jsons[0].get(
-                "first_published_to_production"
+            "first_published_to_production": _get(
+                self.jsons[0], "first_published_to_production"
             ),
-            "last_published_to_production": self.jsons[0].get(
-                "last_published_to_production"
+            "last_published_to_production": _get(
+                self.jsons[0], "last_published_to_production"
             ),
-            "last_unpublishing_date": self.jsons[0].get("last_unpublishing_date"),
+            "last_unpublishing_date": _get(self.jsons[0], "last_unpublishing_date"),
             "retirement_date": self.jsons[0].get("retirement_date"),
```

Comment:
This date still appears as `"None` in the parsed JSON, do we need to change `retirement_date` as well?

Reply:
Oh, good catch. Anything else?